2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_setup.py:_flush():68] Current SDK version is 0.19.1
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_setup.py:_flush():68] Configure stats pid to 964441
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_setup.py:_flush():68] Loading settings from /home/datpd1/.config/wandb/settings
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_setup.py:_flush():68] Loading settings from /home/datpd1/datpd/capstone/Fed_2024/system/wandb/settings
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_setup.py:_flush():68] Loading settings from environment variables
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_init.py:_log_setup():528] Logging user logs to /home/datpd1/datpd/capstone/Fed_2024/system/wandb/run-20250123_142934-brlkxltg/logs/debug.log
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_init.py:_log_setup():529] Logging internal logs to /home/datpd1/datpd/capstone/Fed_2024/system/wandb/run-20250123_142934-brlkxltg/logs/debug-internal.log
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_init.py:init():644] calling init triggers
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_init.py:init():650] wandb.init called with sweep_config: {}
config: {}
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_init.py:init():680] starting backend
2025-01-23 14:29:34,183 INFO    MainThread:964441 [wandb_init.py:init():684] sending inform_init request
2025-01-23 14:29:34,184 INFO    MainThread:964441 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-01-23 14:29:34,185 INFO    MainThread:964441 [wandb_init.py:init():697] backend started and connected
2025-01-23 14:29:34,187 INFO    MainThread:964441 [wandb_init.py:init():790] updated telemetry
2025-01-23 14:29:34,211 INFO    MainThread:964441 [wandb_init.py:init():822] communicating run to backend with 90.0 second timeout
2025-01-23 14:29:34,758 INFO    MainThread:964441 [wandb_init.py:init():874] starting run threads in backend
2025-01-23 14:29:34,812 WARNING MainThread:964441 [wandb_init.py:init():1302] interrupted
Traceback (most recent call last):
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1298, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 915, in init
    run._on_start()
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2414, in _on_start
    self._backend.interface.publish_python_packages(working_set())
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/site-packages/wandb/sdk/interface/interface.py", line 328, in publish_python_packages
    for pkg in working_set:
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/site-packages/wandb/util.py", line 1895, in working_set
    yield InstalledDistribution(key=d.metadata["Name"], version=d.version)
                                                                ^^^^^^^^^
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/importlib/metadata/__init__.py", line 467, in version
    return self.metadata['Version']
           ^^^^^^^^^^^^^
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/importlib/metadata/__init__.py", line 452, in metadata
    return _adapters.Message(email.message_from_string(text))
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/__init__.py", line 37, in message_from_string
    return Parser(*args, **kws).parsestr(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/parser.py", line 64, in parsestr
    return self.parse(StringIO(text), headersonly=headersonly)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/parser.py", line 53, in parse
    feedparser.feed(data)
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/feedparser.py", line 174, in feed
    self._call_parse()
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/feedparser.py", line 178, in _call_parse
    self._parse()
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/feedparser.py", line 238, in _parsegen
    self._parse_headers(headers)
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/feedparser.py", line 486, in _parse_headers
    self._cur.set_raw(*self.policy.header_source_parse(lastvalue))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datpd1/miniconda3/envs/datpd1/lib/python3.12/email/_policybase.py", line 303, in header_source_parse
    return (name, value.rstrip('\r\n'))
                  ^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2025-01-23 14:29:34,822 WARNING MsgRouterThr:964441 [router.py:message_loop():75] message_loop has been closed
